packages/material-ui-styled-engine-sc/src/index.js

```diff
@@ -1,14 +1,18 @@
 import scStyled from 'styled-components';
 
 export default function styled(tag, options) {
+  let scStyledPrepared = scStyled(tag);
+
   if (options) {
-    return scStyled(tag).withConfig({
+    scStyledPrepared = scStyled(tag).withConfig({
```

Comment:
Is there a performance issue with calling `scStyled` twice (first time for nothing?)

Reply:
You are correct, let me refactor to avoid unnecessary call.